packages/ckeditor5-paragraph/src/insertparagraphcommand.ts

```diff
@@ -82,3 +101,12 @@ export default class InsertParagraphCommand extends Command {
 		} );
 	}
 }
+
+// Creates bellow position where should be inserted new paragraph.
+export function createPositionBellow( writer: Writer, position: Position, allowedParent: Element ): Position {
```

Comment:
```suggestion
export function createPositionBelow( writer: Writer, position: Position, allowedParent: Element ): Position {
```

You probably meant that 😉